posix.mak

```diff
@@ -269,12 +269,10 @@ dlangspec.tex : $(DMD) $(DDOC) latex.ddoc dlangspec-consolidated.d
 	$(DMD) -Df$@ $(DDOC) latex.ddoc dlangspec-consolidated.d
 
 # Run twice to fix multipage tables and \ref uses
-dlangspec.dvi : dlangspec.tex
-	latex $^
-	latex $^
-
-$(DOC_OUTPUT_DIR)/dlangspec.pdf : dlangspec.dvi
-	dvipdf $^ $@
+$(DOC_OUTPUT_DIR)/dlangspec.pdf : dlangspec.tex
+	pdflatex -draftmode $^
+	pdflatex $^
+	-mv dlangspec.pdf $(DOC_OUTPUT_DIR)/dlangspec.pdf
```

Comment:
> The result of mv shouldn't be ignored.

It  is to silence the error in the case that `$(DOC_OUTPUT_DIR)` is `.`.



> Better yet, use the -output-directory option of pdflatex to create the result straight at destination.

This will put all the intermediate junk there too.